dali/pipeline/operators/resize/resize_base.cc

```diff
@@ -52,8 +61,8 @@ void ResizeBase::GetResamplingFilters(kernels::FilterDesc &min_filter,
   else if (spec.HasArgument("interp_type"))
```

Comment:
why don't you rely on default arguments instead of initializing to default and then checking if HasArgument?

Reply:
I have to know if the argument has been specified or not. If it has, it overrides the value defined by `interp_type`. It's different than relying on default value, in which case I wouldn't know if the value was specified, but was assigned the default value explicitly, or was not defined at all (in which case interp_type should be used, if defined). The logic is a bit convoluted, but it preserves the old behavior if `interp_type` is set.